### ISSUE TITLE
add an "edit mods" or "view mods" button to either the context menus or the rightside pane

### ISSUE THREAD

**Issue by AmenneHolelane** [PrismLauncher/PrismLauncher#248]
### Role

downloading curseforge packs mainly

### Suggestion

add an "edit mods" or "view mods" button

### Benefit

it's at least one more click than neccessary to add a mod to a pack and i think that other launcher has it

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

i searched "mods" in the issues and didn't find anything. the button would fit well under the edit button, multimc has it [here](https://i.imgur.com/0g1GGoL.png) and it would make downloading curseforge packs with blocked mods easier.

**Comment by DioEgizio** [comment]
i don't quite get this, you can do the same just doing one extra click

**Comment by AmenneHolelane** [comment]
yeah but the option is there on both poly and multimc. Searching in the discord as well results in people asking where those options are/went so it's clearly a feature people used

**Comment by DioEgizio** [comment]
I don't agree, actually they got removed because they were redundant

**Comment by AmenneHolelane** [comment]
make it a configurable option then? maybe add a way to set a default window that the edit window will open to?

**Comment by DioEgizio** [comment]
I don't think it'd make sense and the only possible way would be a submenu, which would still be one more extra click. Honestly I don't get this, it's just a click lol

**Comment by AmenneHolelane** [comment]
set edit window to configurably default to open on view mods is one click i agree. one click and i don't need to do it every time i open that window

**Comment by ghost** [comment]
we don't need those buttons IMO, they are redundant ^^^

but more customization options regarding UI are always welcome

**Comment by 06000208** [comment]
i used those buttons quite frequently, actually (launch offline, view mods folder) and it was the first thing i noticed switching to prism launcher from multimc

i would appreciate them back, or an option to use them again. found this when looking for issue(s) to avoid making a duplicate

#144 asks for the options return, though not in sidebar button form

#253 is somewhat unclear in asking for their removal. from the dropdown, or what? im not sure, since the buttons are already gone, and removing launch offline from the dropdown would make it quite difficult to use

the advantage of having these in the sidebar is a quick 1 click way of launching offline or viewing the instance's specific folders

**Comment by ghost** [comment]
> 
yeah, so... the launch offline option is in the dropdown and the mods folder is just, uh, 2 clicks away at most? it's not worth keeping basically a duplicate of "Edit" on the sidebar.

**Comment by MattHardwick34** [comment]
another vote to bring these options back from me! they are used, people are asking for them, they might seem redundant or just one less click but they are a time saver, it's one of the main arguments i hear from modders/and content creators for not switching from M*ltiMC.

**Comment by maskersss** [comment]
vouch
having to click two more times to get to the config folder, when you do it quite often, can be a bit annoying for me
and i'd imagine basically everyone clicking this button is looking for something in `.minecraft`? so i don't see the point in the button bringing you to its parent folder instead

**Comment by SirGoldalot** [comment]
Vouch.
although they are 'redundant' they are a togglable options.
most of the options in the toolbar actions are redundant and can all be accessed with the toolbars' folder button. we add toolbar actions are there to make life easier, and can be toggled off to reduce clutter.
I personally only have Kill, edit, and folder enabled and I'm sure other users choose to modify their action bar to their use case.
